File: Corrections/php/Neufplate/src/AvatarProvider/SpriteType.php
<?php

namespace App\AvatarProvider;

enum SpriteType: String {
    case MALE = 'male';
    case HUMAN = 'human';
    case FEMALE = 'female';
    case IDENTICON = 'identicon';
    case INITIALS = 'initials';
    case BOTTTS = 'bottts';
    case AVATAAARS = 'avataaars';
    case JDENTICON = 'jdenticon';
    case GRIDY = 'gridy';
    case MICAH = 'micah';
    case DEFAULT = 'default';
}

File: Corrections/java/Neufplate/src/main/java/fr/neufplate/AvatarProvider/SpriteType.java
package fr.neufplate.AvatarProvider;

public enum SpriteType {
    MALE("male"),
    FEMALE("female"),
    HUMAN("human"),
    IDENTICON("identicon"),
    INITIALS("initials"),
    BOTTTS("bottts"),
    AVATAAARS("avataaars"),
    JDENTICON("jdenticon"),
    GRIDY("gridy"),
    MICAH("micah"),
    DEFAULT("default");

    public final String value;

    SpriteType(String value) {
        this.value = value;
    }
}